shared/guide/system/software/sap/var_allowed_users_in_etc_passwd.var

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Extrally Allowed Local Users on the Operating System'
+
+
+description: |-
+    List the users that explicitly allowed on the operating system. This list includes both
+    the users requried by the operating system and the (to be) installed applications.
+    Depending on the Operating System distribution, version, software groups and applications,
+    the user list is different and can be customized with scap-workbench.
+    OVAL reqular expression is used for the user list.
+    <weblink-macro link="http://oval.mitre.org/language/about/re_support_5.6.html" />
+    The list starts with '^' and ends with '$' so that it matches exactly the
+    username, not any string that includes the username. Users are separated with '|'.
+    For example, three users: bin, oracle and sapadm are allowd, then the list is
+    <tt>^(bin|oracle|sapadm)$</tt>. The user <tt>root</tt> is the only user that is hard coded
+    in OVAL that is always allowed on the operating system.
+
+type: string
+
+operator: equals
+
+interactive: true
+
+options:
+    ol7forsap: "^(root|bin|daemon|adm|lp|sync|shutdown|halt|mail|operator|games|ftp|nobody|pegasus|systemd-bus-proxy|systemd-network|dbus|polkitd|abrt|unbound|tss|libstoragemgmt|rpc|colord|usbmuxd$|pcp|saslauth|geoclue|setroubleshoot|rtkit|chrony|qemu|radvd|rpcuser|nfsnobody|pulse|gdm|gnome-initial-setup|postfix|avahi|ntp|sshd|tcpdump|oprofile|uuidd)$"
+    saponol7 : "^(root|bin|daemon|adm|lp|sync|shutdown|halt|mail|operator|games|ftp|nobody|pegasus|systemd-bus-proxy|systemd-network|dbus|polkitd|abrt|unbound|tss|libstoragemgmt|rpc|colord|usbmuxd$|pcp|saslauth|geoclue|setroubleshoot|rtkit|chrony|qemu|radvd|rpcuser|nfsnobody|pulse|gdm|gnome-initial-setup|postfix|avahi|ntp|sshd|tcpdump|oprofile|uuidd|[a-z][a-z0-9][a-z0-9]adm|ora[a-z][a-z0-9][a-z0-9]|oracle|sun|xirui)$"
```

Comment:
Rather than have regex (which we really don't want as variables) it would be better and easier for future users to make this only a comma separated list.